spec/unit/hanami/cli/commands/app/generate/action_spec.rb

```diff
@@ -5,17 +5,22 @@
 require "ostruct"
 
 RSpec.describe Hanami::CLI::Commands::App::Generate::Action, :app do
-  subject { described_class.new(fs: fs, inflector: inflector, generator: generator) }
+  subject { described_class.new(fs: fs, inflector: inflector, generator: action_generator) }
 
   let(:fs) { Dry::Files.new(memory: true) }
   let(:inflector) { Dry::Inflector.new }
-  let(:generator) { Hanami::CLI::Generators::App::Action.new(fs: fs, inflector: inflector) }
+  let(:action_generator) { Hanami::CLI::Generators::App::Action.new(fs: fs, out: stdout, inflector: inflector) }
+  let(:stdout) { StringIO.new }
   let(:app) { Hanami.app.namespace }
   let(:dir) { inflector.underscore(app) }
   let(:controller) { "users" }
   let(:action) { "index" }
   let(:action_name) { "#{controller}.#{action}" }
 
+  def output
```

Comment:
Any better ideas? Can't memoize it as a `let` with how the spec are structured now (particularly the `"infers RESTful action URL and HTTP method for routes"` specs)

Reply:
This is fine